gauriramesh/nvolve2
# Location Form

### Notes
- It occupies the entire epr form right now since conditionally rendering it hasn't been implemented yet.
- I put the form in the middle of the page so that it feels less empty, we can discuss if we'd like to look for another option.

### Test Steps
- Run `nom install` to get the new select package
- Navigate to `http://localhost:3000/epr/nebraskansforcheese`
- Add an on campus location from the list
- Verify that the checkbox appears invalid until checked
- Add an off campus location
- Remove both
